### PR TITLE
Simplify the imports

### PR DESCRIPTION
**Pull request by jasongrout**
In particular, just use the parts of lodash we actually use.

### CODE REVIEW

**Comment by blink1073** [comment]
Nice!

File: packages/controls/src/utils.ts [hunk 1]
```diff
@@ -1,65 +1,15 @@
...
-    let s = [];
-    const hexDigits = '0123456789ABCDEF';
-    for (let i = 0; i < 32; i++) {
-        s[i] = hexDigits.substr(Math.floor(Math.random() * 0x10), 1);
-    }
-    s[12] = '4';  // bits 12-15 of the time_hi_and_version field to 0010
-    s[16] = hexDigits.substr((s[16] & 0x3) | 0x8, 1);  // bits 6-7 of the clock_seq_hi_and_reserved to 01
+import {
+    WrappedError
+} from '@jupyter-widgets/base';
```
Comment (vidartf) [inline packages/controls/src/utils.ts]: Note: This is now imported twice.

Comment (jasongrout) [reply packages/controls/src/utils.ts]: Thanks. See #1808

Comment (jasongrout) [reply packages/controls/src/utils.ts]: Oh wait. it is not imported twice. It's exported once, and imported once.

Comment (vidartf) [reply packages/controls/src/utils.ts]: Oh! Sorry for the noise then!

**Pull request merged**